_data/simple-icons.json

```diff
@@ -12656,6 +12656,11 @@
             "hex": "FCC21B",
             "source": "https://github.com/charliermarsh/ruff/blob/e04ef423344837c916e75a7b09ea674711a104e0/assets/badge/v1.json"
         },
+        {
+            "title": "Rumahweb",
+            "hex": "47CCFF",
```

Comment:
```suggestion
            "hex": "2EB4E3",
```

Reply:
The color should be `2EB4E3`. You can find the resource from the footer of the homepage.

Reply:
Hello @LitoMore I assume you are talking about the colored logo in footer, but in pr #10253 @service-paradis said the hex is there to store the brand color, not the logo color.